src/main/resources/META-INF/reaktivity/mqtt.idl

```diff
@@ -51,7 +52,7 @@ scope mqtt
     {
         struct MqttBeginEx extends core::stream::Extension
         {
-            MqttRole role;
+            MqttCapabilities capabilities;
```

Comment:
Recommend defaulting to `PUBLISH_AND_SUBSCRIBE` as streams are bidirectional by default in reaktivity.